products/ovirt.md

```diff
@@ -0,0 +1,61 @@
+---
+title: oVirt
+category: os
```

Comment:
```suggestion
category: server-app
```

Looking at [the documentation](https://ovirt.org/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.html) oVirt does not look to be an OS.

Reply:
thats true, applied